storybook/stories/Examples/BarChart.stories.tsx

```diff
@@ -92,6 +93,39 @@ export const Stacked = {
   },
 };
 
+const pvErrorData = pageData.map(d => ({ ...d, pvError: [100, 200] }));
```

Comment:
Added this 
```js
const pvErrorData = pageData.map(d => ({ ...d, uvError: [100, 200], pvError: [100, 200] }));
```
and then add an ErrorBar to the first Bar too and this is how it looks, is this expected?

![image](https://github.com/recharts/recharts/assets/41629832/f90f55d7-2272-4190-aaad-bef325115cb9)

Reply:
Ehh, yeah we could make a separate issue for this probably. Most likely this was also what happened before this bug was introduced. At least now we can get error bars on stacked bars at all. In general I think they would mostly be used on the end bars

Reply:
Alright, great work, that was the only concern, will approve.